### DISCUSSION TITLE
New Threats? (Question on Lesson 2: Safe Data Networks for Business, Governance & R&D)

### DISCUSSION THREAD

**Discussion by hakanjonsson** [OpenMined/courses#95] [Course: Our Privacy Opportunity] [answered]
Cynthia Dwork's mentioned on a new threat to privacy that had been applied to census data. Are there any references to this new threat where we can find more information?

**Comment by iamtrask** [comment]
Timestamp?

**Comment by deep1401** [comment, marked as answer]
https://twitter.com/john_abowd/status/1114942180278272000?s=19

Please check if this is what you're looking for!

**Reply by iamtrask** [reply]
That definitely qualifies - whether it's exactly what Cynthia had in mind I can't exactly remember - but that definitely qualifies.

**Reply by hakanjonsson** [reply]
Thanks, that seems to be it. Very interesting.
